README.md

```diff
@@ -45,7 +45,7 @@ Switch the `global.mode` to `"mist"` in the `main.js`.
 
 For development we start the interface with a Meteor server for autoreload etc. Start the interface in a separate terminal window:
 
-    $ cd interface && meteor
+    $ cd mist/interface && meteor
```

Comment:
This was assuming you're already in the mist folder, because the previous steps went in there. Not sure if we would need to show this in each step again. Otherwise i would need to do this in the others as well.. :\